src/composables/use-dialog-content.js

```diff
@@ -30,7 +30,13 @@ export function useDialogContent({ emit, ...props }) {
     visibleRef: props.visibleRef,
     autoFocusOnHideRef: props.autoFocusOnHideRef,
   })
-  useHideOnClickOutside(props)
+  useHideOnClickOutside({
+    dialogRef: props.dialogRef,
+    visibleRef: props.visibleRef,
+    hideOnClickOutsideRef: props.hideOnClickOutsideRef,
+    triggerElementRef: props.triggerElementRef,
+    hideRef: props.hideRef,
```

Comment:
Is this necessary for proper type suggestions or something?

Reply:
It is easier to see which props are passed in this way. There is a problem in one of the composables where the prop can be either a `Ref<HTMLElement|undefined>` or `undefined`, and it was difficult to catch where it was exactly with several layers of abstraction.